frontend/src/pages/home/aiFlows/DeployAndMonitorGallery.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { ProjectObjectType, SectionType, typedObjectImage } from '~/concepts/design/utils';
+import DividedGallery from '~/concepts/design/DividedGallery';
+import InfoGalleryItem from '~/concepts/design/InfoGalleryItem';
+import useServingPlatformStatuses from '~/pages/modelServing/useServingPlatformStatuses';
+
+const DeployAndMonitorGallery: React.FC<{ onClose: () => void }> = ({ onClose }) => {
+  const servingPlatformStatuses = useServingPlatformStatuses();
+  const modelMeshEnabled = servingPlatformStatuses.modelMesh.enabled;
+
+  return (
+    <DividedGallery
+      minSize="225px"
+      itemCount={modelMeshEnabled ? 2 : 1}
```

Comment:
Same operation here -- just so there are no magic numbers that need to be auto synced at the same time